test/webdriverio/customCommands.js

```diff
@@ -1,5 +1,14 @@
 var services = require('./services.conf');
 
+browser.addCommand("waitUrl", function (url, timeout) {
```

Comment:
Page can be redirected to another url by clicking a submit button. During testing this situation can be handled by this command.